### PR TITLE
Version History 5.3.1

### PR DESCRIPTION
**Pull request by dgault**
Updated version history for 5.3.1 release.

Only PR included is https://github.com/openmicroscopy/bioformats/pull/2699#issuecomment-267576295

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Version history
 ===============
 
+5.3.1 (2016 December 16)
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: I think the current date would be Monday (19)

File: docs/sphinx/about/whats-new.txt [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+-----------------------
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: Missing dashes

File: docs/sphinx/about/whats-new.txt [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+
+Java bug fixes:
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: Now we decoupled the low-level components, does this section still make sense /cc @hflynn ?

File: docs/sphinx/about/whats-new.txt (hunk 2 above)
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: I think your underlining here is one character too short

File: docs/sphinx/about/whats-new.txt [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
+
+* TIFF
+   - fixed invalid IFD values when writing TIFF files with ``loci.formats.TiffWriter``
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: Might be worth precising (OME-)TIFF as both formats are affected. Do we need to precise the affected utilities like `bfconvert`?

File: docs/sphinx/about/whats-new.txt (hunk 3 above)
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: I'd be inclined to replace it with file format fixes (or other as appropriate down the line)

**Comment by hflynn** [comment]
👍 looks good

**Pull request merged**